helper/calculateNetworth.js

```diff
@@ -58,7 +58,7 @@ const calculateNetworth = (items, purseBalance, bankBalance, prices, onlyNetwort
 const calculateItemNetworth = (item, prices) => {
   const isPet = item.tag?.ExtraAttributes?.petInfo || item.exp;
   if (isPet) {
-    const petInfo = item.tag?.ExtraAttributes?.petInfo ? JSON.parse(item.tag.ExtraAttributes.petInfo) : item;
+    const petInfo = typeof item.tag?.ExtraAttributes?.petInfo === "object" ? item.tag?.ExtraAttributes?.petInfo : item.tag?.ExtraAttributes?.petInfo ? JSON.parse(item.tag.ExtraAttributes.petInfo) : item;
```

Comment:
we use `'` not `"` lol

Reply:
I can change if u want 🤷‍♂️

Reply:
idk, i feel like alt is right that this change should be on your end not ours. up to him tho